app/models/social_networking/like.rb

```diff
@@ -8,15 +8,54 @@ class Like < ActiveRecord::Base
     # a Participant may only Like an item once
     validates :item_id, uniqueness: { scope: [:item_type, :participant_id] }
 
-    scope :for_today, lambda {
-      where(arel_table[:created_at]
-              .gteq(Date.today.beginning_of_day)
-              .and(arel_table[:created_at].lteq(Date.today.end_of_day)))
-    }
+    scope :for_today,
+          lambda {
+            where(arel_table[:created_at]
+                    .gteq(Date.today.beginning_of_day)
+                    .and(arel_table[:created_at].lteq(Date.today.end_of_day)))
+          }
 
-    scope :for_week, lambda {
-      where(arel_table[:created_at]
-              .gteq(Time.current.advance(days: -7).beginning_of_day))
-    }
+    scope :for_week,
+          lambda {
+            where(arel_table[:created_at]
+                    .gteq(Time.current.advance(days: -7).beginning_of_day))
+          }
+
+    def item_description
```

Comment:
Real Last Question: Should this be included in this story, since it touches additional classes?